browser.js

```diff
@@ -17,11 +17,11 @@ const major = v => v.split('.')[0]
 
 const installing = {
   run: false,
-  start: function () {
+  start: () => {
```

Comment:
This won’t work with `this`; but you’re lucky, there’s an even simpler way that works just fine: `start() {`